security/calico-enterprise/compliance.md

```diff
@@ -0,0 +1,79 @@
+
+![tiered-policy-board]({{site.baseurl}}/images/tiered-policy-board.png)
+
+All Calico Enterprise resources support Kubernetes RBAC permissions, including tiers, and policies within tiers. This enables delegation of trust across teams (e.g. ensuring that the security tier can only be changed by the security team, and that dev team policies are always lower precedence than security team policies).
+
+### Flow logs
+
+To generate proof for your security team that your policies are being enforced and behaving as intended, Calico Enterprise generates and stores flow logs that can be queried to provide reports to your security team and auditors. Accurate flow log data is a common requirement for many compliance frameworks. 
+
+Calico Enterprise flow logs contain important Kubernetes metadata including (but not limited to):
+
+- Source and destination namespaces
+- Source and destination pods
+- Labels for source and destination pods
+- Originating IP address for ingress connections
+- Which policies evaluated the connection and whether the policy allowed or denied the connection
+
+With Calico Enterprise flow logs, can you prove to security and auditors that your policies are in place and being enforced. 
+
+![flow-logs]({{site.baseurl}}/images/flow-logs.png)
+
+### Audit logs
+
+Your security team or auditor may not be satisfied with a “current point in time” compliance audit and may want to see historical data, including what changes have occurred to your policies over time.
+
+Audit logs, when combined with flow logs, demonstrate ongoing compliance with your security requirements over time.
+
+Calico Enterprise maintains a version history for all policy changes including what specifically changed within each policy.
+
+![audit-logs]({{site.baseurl}}/images/audit-logs.png)
+
+### Configuration auditing
+
+Your security team or auditor may also ask about the configuration of your Kubernetes cluster to ensure it aligns with best-practices for security. The industry standard spec for Kubernetes security configuration is the [Center for Internet Security Benchmark for Kubernetes](https://www.cisecurity.org/benchmark/kubernetes/) (CIS).
+
+Calico Enterprise audits the configuration of your cluster and reports on compliance and noncompliance for CIS Level 1 and Level 2 security. Configuration audits are periodically executed, enabling a historical view of the configuration over time. 
+
+![configuration-compliance]({{site.baseurl}}/images/configuration-compliance.png)
+
+The specific findings can be downloaded as a CSV file that provides detailed information about the current value of each configuration setting and the suggested configuration per the CIS standard. If the fix is not viable but is an acceptable risk to take within your organization, you can configure the report specification to exclude that test index so that it no longer shown in the dashboard.
```

Comment:
What is a `test index`?

Reply:
If you look at screen shot, there is a column called "Index"